xla/service/gpu/gpu_compiler.cc

```diff
@@ -1236,14 +1236,22 @@ absl::Status RunAsyncDotPasses(HloModule* hlo_module) {
   return pipeline.Run(hlo_module).status();
 }
 
-absl::Status RunDynamicSliceFusionPasses(HloModule* hlo_module,
-                                         se::Platform::Id platform_id) {
-  if (hlo_module->config()
-          .debug_options()
-          .xla_gpu_enable_dynamic_slice_fusion()) {
+absl::Status RunDynamicSliceFusionPasses(
+    HloModule* hlo_module, se::Platform::Id platform_id,
+    const se::DeviceDescription& device_description, int64_t pointer_size) {
+  const DebugOptions& opts = hlo_module->config().debug_options();
+  if (opts.xla_gpu_enable_dynamic_slice_fusion()) {
     HloPassPipeline pipeline("dynamic-slice");
     TF_ASSIGN_OR_RETURN(se::Platform * platform,
                         se::PlatformManager::PlatformWithId(platform_id));
+    pipeline.AddPass<GpuReduceScatterCombiner>(
+        device_description, /*default_combine_threshold_in_bytes=*/
+        kDefaultReduceScatterCombineThreshold,
+        /*combine_threshold_in_bytes=*/
+        opts.xla_gpu_reduce_scatter_combine_threshold_bytes(),
+        /*combine_threshold_count=*/256,
```

Comment:
Why 256? Can you extract this as a constant?

Reply:
Done.